descriptors/ModuleDescriptor-template.json

```diff
@@ -545,6 +554,25 @@
         "acquisitions-units.units.item.delete"
       ]
     },
+    {
+      "permissionName": "orders.acquisitions-units-assignments.item.post",
+      "displayName": "Acquisitions unit assignment - create unit assignment",
+      "description": "Create a new acquisitions unit assignments"
```

Comment:
Minor: Descriptions and display name for these permissions can be changed a bit to reflect that the assignment is being added to order